File: Clientside/Admin/src/models/OrderDto.ts
import BaseOrderDto from "./BaseOrderDto";
import VariantOrderDto from "./VariantOrderDto";

interface OrderDto extends BaseOrderDto {
    firstName: string
    lastName: string
    address: string
    town: string
    zipCode: string
    variantOrders: VariantOrderDto[]
}

export default OrderDto;

File: Serverside/Admin/Api/Dtos/Order/OrderDto.cs
using System.Collections.Generic;

namespace FNO.Admin.AdminApi.Dtos.Order
{
    public class OrderDto : BaseOrderDto
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Town { get; set; }
        public int ZipCode { get; set; }
        public List<VariantOrderDto> VariantOrders { get; set; }
    }
}
